release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1032,7 +1032,7 @@ The 1.6.2 version of the Network Observability Operator fixes a compatibility is
 [id="ocp-4-17-etcd-4-5-nodes_{context}"]
 ==== Node scaling for etcd
 
-In this release, if your cluster is installed on a bare metal platform, you can scale a cluster to up to 5 nodes as a post-installation task. The etcd Operator scales accordingly to account for the additional node. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-node-scaling_recommended-etcd-practices[Node scaling for etcd].
+In this release, if your cluster is installed on a bare metal platform, you can scale a cluster control plane up to 5 nodes as a post-installation task. The etcd Operator scales accordingly to account for the additional control plane nodes. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-node-scaling_recommended-etcd-practices[Node scaling for etcd].
```

Comment:
Same out of scope suggestion about the postinstall phrasing as your other PR:
```suggestion
In this release, if your cluster is installed on a bare metal platform, you can scale a cluster control plane up to 5 nodes as a postinstallation task. The etcd Operator scales accordingly to account for the additional control plane nodes. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-node-scaling_recommended-etcd-practices[Node scaling for etcd].
```

Reply:
Fixed